openwisp_utils/admin_theme/admin.py

```diff
@@ -20,3 +22,22 @@ def openwisp_admin(site_url=None):
                 'OPENWISP_ADMIN_INDEX_TITLE',
                 'Network administration')
     )
+
```

Comment:
change also `openwisp_admin` so that we avoid the duplication of lines like:

`site_title = getattr(settings, # ...`.

Move the function below, and use the class attributes defined in `OpenwispAdminSite` instead of redefining them.

Add a docstring to the function explaining that its usage it's deprecated.